test/steps/setup.js

```diff
@@ -186,6 +192,9 @@ const restoreEnvVariable = variable =>
 			delete process.env[variable];
 		}
 	};
+export function setUpLiskJSAPIInstance() {
```

Comment:
The exported functions should be named specifically for the suite. I.e. `setUpUtilQuery`. And we could do with a blank line before this function.

Reply:
for testing `Query Class` suite, it would be like `setUpQueryClass`?

Reply:
We use `setUpUtilCrypto` even though that's a class. I would say class is an implementation detail, and the name should be about the module.